src/magic_folder/config.py

```diff
@@ -118,10 +126,60 @@
         )
         """,
         """
-        CREATE TABLE local_snapshots
+        CREATE TABLE [local_snapshots]
         (
-            path          TEXT PRIMARY KEY,  -- the (mangled) name in UTF8
-            snapshot_blob BLOB               -- a JSON blob representing the snapshot instance
+            -- A random, unique identifier for this particular snapshot.
+            [identifier]       TEXT PRIMARY KEY,
+
+            -- The magicpath-mangled name of the file this snapshot is for,
+            -- UTF-8-encoded.
+            [path]             TEXT,
```

Comment:
basically bike-shed-ing here, but maybe `name` would be better? (We use "name" everywhere else to describe the flattened snapshot name)

Reply:
Sure.  Consistency is good.  fwiw, "path" makes more sense to me than "name" but since it's name in a lot of other places I'll adjust the database to match that.